src/api/config/application.rb

```diff
@@ -15,6 +15,7 @@ class Application < Rails::Application
 
     # Custom directories with classes and modules you want to be autoloadable.
     # config.autoload_paths += %W(#{config.root}/extras)
+    config.eager_load_paths << Rails.root.join('lib')
```

Comment:
Uhm there is so much more stuff in lib. Have you checked that this is okay for everything?